### ISSUE TITLE
Cross-file "problems" never clear (until vscode restart)

### ISSUE THREAD

**Issue by thelazydogsback** [ionide/ionide-vscode-fsharp#1263]
Any symbol resolution problems that are cross-file are never cleared in Problems (or the editor squigglies) until vscode is restarted. (Save all & full rebuild does not help.)

**Comment by ianvox** [comment]
I'm completely new to using Ionide for F# projects (have been using it to write/run FSX scripts for a while), but I'll throw in my two cents:

I've had the same issue when the project is not part of a solution. Seems I can work around it by adding the project to a solution and re-opening (`dotnet new sln` to create the .sln file, then `dotnet sln [path to .sln file] add [path to .fsproj file]`). Sometimes I have to rebuild the solution, then restart OmniSharp (both of these via VS Code Command Palette). Sometimes I can get away with deleting the solution file afterward, sometimes not. In any case, the squiggles don't go away until I've edited the file (e.g. insert a space somewhere).

Could this be related to https://github.com/ionide/ionide-vscode-fsharp/issues/917 ?

In case it would be helpful, here's some diagnostic info about my machine:
## Machine infos

* Operating system: **Windows_NT**
* Arch: **x64**
* VSCode: **1.41.1**
* Runtime: **netcore**
* Dotnet version: **3.1.100**
* MSBuild version:
```shell
Microsoft (R) Build Engine version 4.8.3752.0
[Microsoft .NET Framework, version 4.0.30319.42000]
Copyright (C) Microsoft Corporation. All rights reserved.
```

**Comment by Krzysztof-Cieslak** [comment]
This should be fixed from `4.4.4`
